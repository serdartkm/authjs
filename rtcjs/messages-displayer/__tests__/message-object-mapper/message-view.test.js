import React from 'react'
import renderer from 'react-test-renderer'
import MessageView from '../../lib/message-object-mapper/message-view'

describe("MessageView snapshot test",()=>{

    it('renders correctly', () => {
        const tree = renderer
            .create(<MessageView message="hello from subsequent message" datetime={1569393960355} />).toJSON()

        expect(tree).toMatchSnapshot();
    })
})