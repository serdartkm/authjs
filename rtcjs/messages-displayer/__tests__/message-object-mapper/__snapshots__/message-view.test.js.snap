// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageView snapshot test renders correctly 1`] = `"<div style=\\"padding: 5px; margin: 2px; border-radius: 15px; border-color: #9E9E9E; border-style: solid; border-width: 2px; max-width: 100%; word-wrap: break-word; word-break: break-all; min-width: 30%;\\"><div>hello from subsequent message</div><div style=\\"font-size: 10px; padding-top: 2px; text-align: end;\\"><i style=\\"background-color: #efebe9;\\">11:46:00 AM</i></div></div>"`;
